src/importlinter/contracts/independence.py

```diff
@@ -19,19 +19,28 @@ class IndependenceContract(Contract):
         - ignore_imports: A set of ImportExpressions. These imports will be ignored: if the import
                           would cause a contract to be broken, adding it to the set will cause
                           the contract be kept instead. (Optional.)
+        - unmatched_ignore_imports_alerting: Decides how to report when the expression in the
+                          `ignore_imports` set is not found in the graph. Valide values are
+                          "none", "warn", "error". Default value is "error".
     """
 
     type_name = "independence"
 
     modules = fields.ListField(subfield=fields.ModuleField())
     ignore_imports = fields.SetField(subfield=fields.ImportExpressionField(), required=False)
+    unmatched_ignore_imports_alerting = fields.StringField(required=False)
 
     def check(self, graph: ImportGraph) -> ContractCheck:
         is_kept = True
         invalid_chains = []
+        alerting_level = helpers.parse_unmatched_ignore_imports_alerting(
```

Comment:
I think we should make the `Field` responsible for validating the user input, rather than doing it in this helper. That's what fields are for.

A nice way to do this might be to create a new kind of field called an `EnumField`, which could be used like this:

```
unmatched_ignore_imports_alerting = fields.EnumField(
    enum=AlertLevel,
    default=AlertLevel.ERROR
)
```

What do you think?